apps/design-system-docs/src/components/HomepageFeatures/index.tsx

```diff
@@ -9,42 +9,27 @@ type FeatureItem = {
   description: ReactNode
 }
 
-import DocusaurusMountain from '@site/static/img/undraw_docusaurus_mountain.svg'
-import DocusaurusTree from '@site/static/img/undraw_docusaurus_tree.svg'
-import DocusaurusReact from '@site/static/img/undraw_docusaurus_react.svg'
+import Designer from '../../../static/img/illustrations/designer_illu.svg'
+import Devices from '../../../static/img/illustrations/devices_illu.svg'
+import Puzzle from '../../../static/img/illustrations/puzzle_illu.svg'
 
 const FeatureList: FeatureItem[] = [
   {
     title: 'Resources',
-    Svg: DocusaurusMountain,
-    description: (
-      <>
-        Provides teams with easy-to-use resources, from React & Figma component
-        libraries to icons, design tokens, colors, typography and more
-      </>
-    ),
+    Svg: Puzzle,
+    description: <> React/Figma components, tokens, icons, color palettes.</>,
```

Comment:
🙈 It's a little difficult to understand. Perhaps we can have it like this or something similar?
```suggestion
    description: <> Comprehensive resources like React and Figma components, tokens, icons and colour palettes.</>,
```